test/verify/check-machines-dbus

```diff
@@ -726,9 +727,36 @@ class TestMachinesDBus(machineslib.TestMachines):
             name="my_dir_pool_two",
             pool_type="netfs",
             target="/tmp/my_dir_pool_two/",
-            source={"host": "127.0.0.1", "dirpath": "/mnt/exports"}
+            source={"host": "127.0.0.1", "source_path": "/mnt/exports"}
         ).execute()
 
+        # Debian and ubuntu images don't have open-iscsi already installed
+        # Let's not add extra dependency only for this part of the test.
```

Comment:
This should get a "FIXME:" prefix. Let's add open-iscsi to these images (in a follow-up).

Reply:
Done